tests/test_match.py

```diff
@@ -145,9 +154,11 @@ def test_ne_equal(sample):
             pass
     assert not istr(original) != munged
     assert not istr(original) != istr(munged)
+    for non_string in non_strings:
+        assert istr(original) != non_string
```

Comment:
Hello @gsakkis,
I don't think this loop should be in this particular test since:
  * it performs exactly the same comparison as for `test_ne_different`, so it won't increase the coverage
  * it checks completely opposite to what a test name means
```suggestion
```

Reply:
Thanks @Jamim, i removed the redundant checks.